node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -62,11 +68,45 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val nodeDiscoveryEffect: NodeDiscovery[Task]   = effects.nodeDiscovery[Task](net)
   implicit val transportLayerEffect: TransportLayer[Task] = effects.transportLayer[Task](net)
 
+  val bondsFile: Option[File] =
+    conf.bondsFile.toOption
+      .flatMap(path => {
+        val f = new File(path)
+        if (f.exists) Some(f)
+        else {
+          Log[Task].warn(
+            s"Specified bonds file $path does not exist. Falling back on generating random validators."
```

Comment:
Is generating random validators just for testing?

Reply:
Yes.